ui/nuxeo-document-permissions/nuxeo-popup-permission.js

```diff
@@ -284,6 +288,12 @@ import '../nuxeo-button-styles.js';
       this._doSend(true);
     }
 
+    doCancel() {
+      // get called when clicked on cancel button
```

Comment:
No need to include comments when the code is self-explanatory.

Reply:
removed comment